extensions/amp-a4a/0.1/test/test-a4a-integration.js

```diff
@@ -90,19 +91,25 @@ describe('integration test: a4a', () => {
   beforeEach(() => {
     sandbox = sinon.sandbox.create();
     xhrMock = sandbox.stub(Xhr.prototype, 'fetch');
+    // Expect key set fetches for signing services.
+    const fetchJsonMock = sandbox.stub(Xhr.prototype, 'fetchJson');
+    for (const serviceName in signingServerURLs) {
```

Comment:
I've consulted the JavaScript style authorities at Google and they say this is a matter of individual engineer discretion (although various other people opined that avoiding `Object.keys` because of the array allocation is premature optimization, which it kind of is). Up to you, Keith.